src/limits.cpp

```diff
@@ -1428,8 +1478,7 @@ void misc_update(void)
     }
 
     if (!IS_NPC(ch)) {
-      if (!CH_IN_COMBAT(ch))
-        check_adrenaline(ch, 0);
+      check_adrenaline(ch, 0);
```

Comment:
Why was this taken out of its in-combat guard?

Reply:
Because the in-combat call was removed entirely and this now controls the pump entirely. The previous system with two calls running in different intervals/conditions made controlling pump duration too erratic (It was way to short finishing immediately most of the time). This allows a finer control on the pump duration. Again time abstraction issues here. It now last a lot longer but not sure if the duration I set it to is ok (It's a random roll of 1d6 per level of the pump.. (so max 2d6) I multiplied the duration by 10, it's more usable now but the possible duration range is large).